### DISCUSSION TITLE
Update ListView

### DISCUSSION THREAD

**Discussion by beppo-ivel** [appcelerator/titanium_mobile#13607] [Q&A] [answered]
I have a window with a ListView that shows the records of a collection. And I have a second view which contains another window. The controller that belongs to the second view creates a model that belongs to the collection of the ListView. After closing the the second window the user is back on the first window with the ListView. But the ListView does not show the new record, because I have to pass my new set of items via `$.listView.sections[0].setItems(modelTemplates);`. But `$.listView` is part of the first view/controller and I don't know how to trigger/call it from my second controller.

**Comment by m1ga** [comment, marked as answer]
Backbone events (https://blog.axway.com/product-insights/discontinued/titanium/global-app-events-in-titanium-using-backbone-js) or pass a method in the `createController` when creating the 2nd window.
